prisma/seed.js

```diff
@@ -0,0 +1,21 @@
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+async function main() {
+  const insertResult = await prisma.food.createMany({
+    data: [
+      { id: 10, title: "toast", img: "bellablue", author: "lenovo" },
+      { id: 22, title: "sausage", img: "pinkpurple", author: "lenovo" },
+    ],
+  });
+  console.log(insertResult.count);
```

Comment:
Consider removing debugging code from code destined for the trunk.  Alternatively, if this isn't debugging code, consider making this more robust.  For example:

```suggestion
  console.info(`The prisma database was seeded with ${insertResult.count} items.`);
```

Reply:
Ok will do moving forward...Have the habit of console.logging